src/App.tsx

```diff
@@ -15,71 +19,95 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    selectedGood: 'Jam',
+  };
+
+  handleAddButton = (good: string) => {
+    this.setState({
+      selectedGood: good,
+    });
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handleRemoveButton = () => {
+    this.setState({
+      selectedGood: '',
+    });
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  render() {
+    const { selectedGood } = this.state;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    return (
+      <main className="section container">
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+        <h1 className={`title${selectedGood && ' is-flex is-align-items-center'}`}>
```

Comment:
Usually we use [classnames](https://www.npmjs.com/package/classnames) library for that purpose